UC-Berkeley-Extension/global
# Do we want to use AngularJS in Drupal

## Asana Ticket(s)
- See ticket [Do we want to use AngularJS in Drupal](https://app.asana.com/0/1174970560648905/1194018677557889/f).

## Description
- Adds the [PDB](https://www.drupal.org/project/pdb/) module for Angular blocks.
- Enables a sample block.

## Testing instructions

* Checkout this branch
* From project root, run `composer install`
* Check that the `pdb` module downloaded properly.
* `cd web/modules/contrib/pdb/modules/pdb_ng2`
* `npm install`
* `ddev ssh`
* `drush cr`
* `drush cim`
* Go to https://global.ddev.site/ 
 * Observe the `Voices Post List` block is displayed and has content.

## Screenshots

![Screen Shot 2020-09-29 at 5 08 59 PM](https://user-images.githubusercontent.com/360988/94615948-97234680-0276-11eb-9fe3-50d230b6dca7.png)


## Remaining tasks

- [x] Take existing angular code and try to use it in a custom block
